### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by gucio321**

### CODE REVIEW

**Comment by paladium** [comment]
Can I merge the PR, because github said it is still in draft?

**Comment by gucio321** [comment]
I've a few questions:
- what do you need to identity a bug with CI? (build URL, project ID...)
- do you want to use GH discussions for questions?
- do you allow blank issues?

**Comment by gucio321** [comment]
hmm, I just found this: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
as for issues-only repo, it could be very cool

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+#  - name: Question
+#    about: If you have a question, ask it in a discussion
+#    url: https://github.com/nativeci/issues/discussions
```
Comment (gucio321) [inline .github/ISSUE_TEMPLATE/config.yml:5]: if you want to use discussions for questions like here:
image

please enable them in settings, else just let me know, and I'll remove these lines

Comment (paladium) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Wow great stuff, I have enabled the discussions

**Comment by paladium** [comment]
Are you ready to merge now?

**Comment by gucio321** [comment]
yah, you can review and merge

**Pull request merged**